### DISCUSSION TITLE
Using server.js to create an api

### DISCUSSION THREAD

**Discussion by mbuguanewton** [jaredpalmer/razzle#1576] [Q&A] [answered]
Hello, i have been trying out razzle for the last couple of days and i wanted to know if using the provided express server can be used to create an api or do you have to create another server for that? For the most part it works but if you make changes on any file import in the server.js you need to restart the application.

**Comment by fivethreeo** [comment, marked as answer]
https://github.com/jaredpalmer/razzle/tree/master/examples/with-typeorm-graphql

**Reply by mbuguanewton** [reply]
Awesome thanks
